catalog/validations/plant-validations.js

```diff
@@ -0,0 +1,34 @@
+const { body } = require('express-validator');
+
+const Plant = require('../models/plant');
+
+const plantValidationRules = () => {
+  const fieldToValidate = ['species', 'plantationType', 'plantationDate', 'harvestDate'];
```

Comment:
What's the point of validating `harvestDate` since it's not mandatory in the model?